apps/frontend/src/hooks/useChainStore.ts

```diff
@@ -18,8 +18,9 @@ export const useChainStore = create<ChainStore>()(
       currentChainId: DEFAULT_CHAIN_ID,
       setCurrentChainId: async (chainId: ChainId = DEFAULT_CHAIN_ID) => {
         // todo: should disconnect wallets which does not support network changes
-        await onboard.setChain({ chainId });
-        set({ currentChainId: chainId });
+        if (await onboard.setChain({ chainId })) {
+          set({ currentChainId: chainId });
+        }
```

Comment:
Leave it like it was - even if user did not change network in their wallet, app still should change context to another wallet. Most of the wallets always returns false in any case.

Modal will appear asking user to change chain in their wallet.
```suggestion
        await onboard.setChain({ chainId }));
        set({ currentChainId: chainId });
```